src/rgw/driver/rados/rgw_sal_rados.cc

```diff
@@ -2551,6 +2552,111 @@ int RadosObject::transition(Bucket* bucket,
                                            mtime, olh_epoch, dpp, y, flags & FLAG_LOG_OP);
 }
 
+int RadosObject::restore_obj_from_cloud(Bucket* bucket,
+                                  rgw::sal::PlacementTier* tier,
```

Comment:
yesterday while fixing tier config issue, I was wondering whether we need both tierconf and Placementtier is required for this function

Reply:
right..I was wondering the same.. now that tier_config is updated with latest Zonegroup tier info, we can avoid this second argument.